src/components/LibraryContainer.tsx

```diff
@@ -69,6 +71,29 @@ export class LibraryContainer extends React.Component<LibraryContainerProps, Lib
         };
     }
 
+    componentWillMount() {
+        window.addEventListener("keydown", this.handleKeyDown.bind(this));
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener("keydown", this.handleKeyDown.bind(this));
+    }
+
+    handleKeyDown(event: any) {
```

Comment:
Can we handle the event in one place? By any chance, if the event handler in SearchResultItem is handled before this one in LibraryContainer, there will be logic error.